assets/src/editor/blocks/carousel/block.json

```diff
@@ -34,6 +39,10 @@
       "type": "boolean",
       "default": true
     },
+    "layout": {
+      "type": "string",
+      "default": "carousel-groups"
```

Comment:
Might be good to put an [enum](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-attributes/#enum-validation) on this